File: Templates/Examples/NHibernate/CSharp/NHibCsSample.Data/ManagerObjects/CartManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using NHibCsSample.Generated.BusinessObjects;
using NHibCsSample.Generated.Base;

namespace NHibCsSample.Generated.ManagerObjects
{
    public partial interface ICartManager : IManagerBase<Cart, int>
    {
	}
	
	partial class CartManager : ManagerBase<Cart, int>, ICartManager
    {
	}
}

File: trunk/Templates/Examples/NHibernate/VisualBasic/NHibVbSample.Data/ManagerObjects/CartManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports NHibVbSample.Generated.BusinessObjects
Imports NHibVbSample.Generated.Base

Namespace NHibVbSample.Generated.ManagerObjects
	'Public Partial Interface ICartManager
	'	Inherits IManagerBase(Of Cart, System.Int32)
	'End Interface

	Partial Class CartManager
		Inherits ManagerBase(Of Cart, System.Int32)
		Implements ICartManager
	End Class
End Namespace
